projects/api/src/lib/services/tailormap-api-v1.service.ts

```diff
@@ -90,11 +90,17 @@ export class TailormapApiV1Service implements TailormapApiV1ServiceModel {
       });
   }
 
-  public deleteFeature$(params: { applicationId: string; layerId: string; feature: FeatureModel }): Observable<HttpStatusCode> {
-    return this.httpClient.delete<HttpStatusCode>(
-      `${TailormapApiConstants.BASE_URL}/${params.applicationId}/layer/${params.layerId}/edit/feature/${params.feature.__fid}`,
-    );
-  }
+    public deleteFeature$(params: { applicationId: string; layerId: string; feature: FeatureModel }): Observable<HttpStatusCode> {
+        return this.httpClient.delete<HttpResponse<Response>>(
+            `${TailormapApiConstants.BASE_URL}/${params.applicationId}/layer/${params.layerId}/edit/feature/${params.feature.__fid}`,
+            { observe: 'response' },
+        ).pipe(
+            map(response => {
+                    return response.status as HttpStatusCode;
```

Comment:
`... as HttpStatusCode` is probably an uneeded cast, `.status` should already be this type?

Reply:
`HttpResponseBase.status` is a number